pkg/metricsstore/metricsstore.go

```diff
@@ -185,6 +191,20 @@ func init() {
 			Help:      "Histogram to track time spent to issue xds certificate",
 		},
 		[]string{})
+
+	/*
+	* Log metrics
+	 */
+	defaultMetricsStore.LogLvlCount = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: metricsRootNamespace,
+			Subsystem: "log",
+			Name:      "level_count",
+			Help:      "Represents the number of logs per log level emitted",
+		},
+		[]string{
+			"logLevel",
```

Comment:
```suggestion
			"log_level",
```
Nit: To be consistent with other metrics